src/program/config/set_operator_state/README

```diff
@@ -0,0 +1,19 @@
+Usage: snabb config set-operator-state [OPTION]... ID ALARM-ID OPERATOR-STATE ALARM-TEXT
```

Comment:
I find "snabb config set-operator-state" a little missing on the fact that it relates to alarms, and I almost wish "alarm" were somehow in the command line somewhere; `snabb config set-operator-alarm-state` or so.  Dunno, if you agree then maybe we should make this change.